website/data/internals-nav-data.json

```diff
@@ -1,5 +1,7 @@
 [
   { "heading": "Terraform Internals" },
+  {
+    "title": "Overview", "path": ""},
```

Comment:
I'd put "path" on its own line like the other sections, keeping it consistent makes it easier to scan the file later.